controllers/managedocs_controller_test.go

```diff
@@ -531,7 +523,7 @@ var _ = Describe("ManagedOCS controller", func() {
 				deployerCSV.Name = testDeployerCSVName
 				deployerCSV.Namespace = testPrimaryNamespace
 				deployerCSV.Spec.InstallStrategy.StrategyName = "test-strategy"
-				deployerCSV.Spec.InstallStrategy.StrategySpec.DeploymentSpecs = []opv1a1.StrategyDeploymentSpec{}
+				deployerCSV.Spec.InstallStrategy.StrategySpec.DeploymentSpecs = getMockOCSOSDDeployerCSVDeploymentSpec()
```

Comment:
Why do we need to refer to a struct that lives in another file? Why can't we just embed the necessary struct here?